src/NServiceBus.AcceptanceTests/Core/OpenTelemetry/Metrics/When_payload_contains_multiple_logical_messages.cs

```diff
@@ -1,59 +0,0 @@
-namespace NServiceBus.AcceptanceTests.Core.OpenTelemetry.Metrics;
```

Comment:
We have no Core acceptance test that covers the scenario of multiple tests, right? Is it reasonable to move this test somewhere in the Core folder to have at least some coverage from an acceptance test level too?